packages/rlc-common/src/buildObjectTypes.ts

```diff
@@ -363,24 +363,34 @@ function getImmediateParentsNames(
 
   const extendFrom: string[] = [];
 
-  // If an immediate parent is a DictionarySchema, that means that the object has been marked
+  // If an immediate parent is an empty DictionarySchema, that means that the object has been marked
   // with additional properties. We need to add Record<string, unknown> to the extend list and
-  if (objectSchema.parents.immediate.find(isDictionarySchema)) {
+  if (
```

Comment:
> with additional properties. We need to add Record<string, unknown> to the extend list and
Just curious the comment says that the additional property is supported in modular?

Reply:
Just notice this is rlc code.